src/main/java/de/tsystems/mms/apm/performancesignature/model/CredProfilePair.java

```diff
@@ -70,12 +67,32 @@ public String getDisplayName() {
             return "";
         }
 
-        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath final Project project) {
-            return new StandardListBoxModel()
-                    .withEmptySelection()
-                    .withMatching(instanceOf(UsernamePasswordCredentials.class),
-                            CredentialsProvider.lookupCredentials(
-                                    StandardUsernameCredentials.class, project, ACL.SYSTEM, Collections.<DomainRequirement>emptyList()));
+        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Jenkins context, @QueryParameter String credentialsId) {
```

Comment:
the CredProfilPair is used in the global configuration of Jenkins, there is no project available and `TopLevelItem ` is null
Is there any object that is available in the global configuration?